addons/website/views/website_views.xml

```diff
@@ -555,12 +555,10 @@
             <field name="model">ir.module.module</field>
             <field name="mode">primary</field>
             <field name="arch" type="xml">
-                <form create="false" edit="false" delete="0" js_class="theme_preview_form">
+                <form create="false" edit="false" delete="0" js_class="theme_preview_form" class="o_view_form_theme_preview">
                   <div class="o_preview_frame h-100">
-                        <field name='url' widget='iframe'/>
-                        <img alt='phone' class='img_mobile' style='display:none' src="/website/static/src/img/phone.png"/>
+                        <field name='url' widget='iframe' class="d-block m-0 h-100"/>
```

Comment:
While you are touching this line, take the opportunity to change the `'` into `"` :+1: